modules/openapi-generator/src/main/resources/go/client.mustache

```diff
@@ -420,8 +420,20 @@ func (c *APIClient) decode(v interface{}, b []byte, contentType string) (err err
 		*s = string(b)
 		return nil
 	}
+	if f, ok := v.(*os.File); ok {
+        f, err = ioutil.TempFile("", "HttpClientFile")
+        if err != nil {
+            return
+        }
+        _, err = f.Write(b)
+        if err != nil {
+            return
+        }
+        _, err = f.Seek(0, io.SeekStart)
+        return
+    }
```

Comment:
Some indentions in this files and others are using 4-space instead of tabs. I'll file another PR to update that.

Reply:
UPDATE: filed https://github.com/OpenAPITools/openapi-generator/pull/14189